tests/unit/amo/reducers/testReviews.js

```diff
@@ -155,49 +155,8 @@ describe(__filename, () => {
     }).toThrow(/review ID 3 .* does not exist/);
   });
 
-  it('preserves existing user rating data', () => {
-    let state;
-
-    state = reviewsReducer(
-      state,
-      setFakeReview({
-        id: 1,
-        userId: 1,
-        addonId: 1,
-        rating: 1,
-      }),
-    );
-
-    state = reviewsReducer(
-      state,
-      setFakeReview({
-        id: 2,
-        userId: 1,
-        addonId: 2,
-        rating: 5,
-      }),
-    );
-
-    state = reviewsReducer(
-      state,
-      setFakeReview({
-        id: 3,
-        userId: 2,
-        addonId: 2,
-        rating: 4,
-      }),
-    );
-
-    // Make sure all reviews co-exist by userId, addonId, review ID.
-    expect(state[1][1][1].rating).toEqual(1);
-    expect(state[1][2][2].rating).toEqual(5);
-    expect(state[2][2][3].rating).toEqual(4);
-  });
-
   it('preserves existing add-on reviews', () => {
```

Comment:
it seems like we have 2 test cases with this same name.  or should we keep both but clarify a bit more?